tests/end_to_end/candid_rpc/class_syntax/list_of_lists/src/index.ts

```diff
@@ -153,7 +153,7 @@ export default class {
 
     @query(
         [IDL.Vec(IDL.Vec(IDL.Vec(IDL.Int64)))],
-        IDL.Vec(IDL.Vec(IDL.Vec(IDL.Nat64)))
+        IDL.Vec(IDL.Vec(IDL.Vec(IDL.Int64)))
```

Comment:
This is interesting, I'm guessing it's because every time it gets serded it will end up a bigint so we never ran into problems?

Reply:
Our hypothesis is that because of bigints in js, this didn't end up being an actual issue.